src/test/user.spec.js

```diff
@@ -192,3 +200,139 @@ describe('Test user login, signup and account verification', () => {
     });
   });
 });
+
+describe('USER CAN FOLLOW OTHER USERS', () => {
+  beforeEach(async () => {
+    await db.User.destroy({ cascade: true, truncate: true });
+  });
+
+  it('should be able to follow other users', async () => {
+    const firstUser = await createUser(await getUser());
+    const secondUser = await createUser(await getUser());
+    await createUser(await getUser());
+    const token = jwt.sign(firstUser, process.env.JWT_SECRET);
+
+    const res = await server()
+      .patch(`${users}/follow`)
+      .set('token', token)
+      .send({ followeeId: secondUser.id });
+    expect(res.status).to.equal(200);
```

Comment:
Test that the second user actually got a new follwer whose Id === firstUser.id

Reply:
Changes effected thank you